tests/rust/common/runtime.rs

```diff
@@ -88,15 +88,21 @@ impl NetworkRuntime for SharedDummyRuntime {
         if let Some(body) = pkt.take_body() {
             buf[header_size..].copy_from_slice(&body[..]);
         }
-        self.outgoing.try_send(buf).unwrap();
+        match self.outgoing.try_send(buf) {
```

Comment:
This failed due to a runtime error. In my opinion, we should fail with `EAGAIN`.